rules/15.fuzzing_payload.xml

```diff
@@ -0,0 +1,46 @@
+<beginning>
+<embedded_functions><![CDATA[
+#include "../lib/http2.h"
+#include <stdlib.h>
+#include <unistd.h>
+
+
+static unsigned long int number_request =1000;
+	void on_load(){
+		
+
+		printf("Loaded successfully rule Fuzzing  \n" );
+		
+	}
+	
+	static void em_modif_then_forward(
+	   const rule_info_t *rule, int verdict, uint64_t timestamp, 
+	   uint64_t counter, const mmt_array_t * const trace ){	
+  	const char* trace_str = mmt_convert_execution_trace_to_json_string( trace, rule );
+   	printf( "FUzzing Update %d\n%s\n", rule->id, trace_str );
+
+   	printf( "em_modif_then_forward \n");
+	for(int i=1;i<number_request;i+=2){
+			set_numeric_value( PROTO_HTTP2, HTTP2_PAYLOAD_FUZZ, i);
+     			forward_packet();
+		}
+     	
+	}
+
+
+
+	void on_unload(){
+		printf("Unloaded successfully rule 1\n");
+	}
+
+]]></embedded_functions>
+<property value="THEN" property_id="15" type_property="FORWARD" 
+    description="Modify stream id" if_satisfied="em_modif_then_forward">
+  <event value="COMPUTE" event_id="1" description="Fuzz payload "
+         		  boolean_expression="( (http2.header_method == 131 ) " />
```

Comment:
Can you please add a condition `ip.src != ip.dst` in this boolean expression? So that the IP addresses will be shown GUI of MMT-Operator.